sdk/core/Azure.Core/tests/MultipartTests.cs

```diff
@@ -123,7 +123,11 @@ public async Task ParseBatchChangesetResponse(string batchResponse)
                 Assert.That(contentType, Is.EqualTo("application/json;odata=fullmetadata;streaming=true;charset=utf-8"));
 
                 var bytes = new byte[response.ContentStream.Length];
+#if NET6_0_OR_GREATER
```

Comment:
[CA2022: Avoid inexact read with Stream.Read](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2022) ReadAsync may return fewer bytes than requested which can result in unreliable code. ReadExactlyAsync should be used instead but it's only available for .NET6+. Since this is a test case, just leave the net462 version as-is.